app/Models/Competition.php

```diff
@@ -31,16 +31,19 @@ public function users()
     /**
      * Gets a CSV export of all the users in this competition.
      *
+     * @param \Gladiator\Repositories\UserRepositoryContract $repository
      * @return \League\Csv\ $csv
      */
-    public function getCSVExport()
+    public function getCSVExport($repository)
```

Comment:
maybe @DFurnes has some input from his experience in Northstar & Aurora?

![image](https://cloud.githubusercontent.com/assets/105849/13825174/bb9942c2-eb87-11e5-9590-0588357dcc43.png)